apps/deploy-web/src/pages/api/bitbucket/authenticate.ts

```diff
@@ -0,0 +1,23 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import BitbucketAuth from "@src/services/auth/bitbucket.service";
+
+const NEXT_PUBLIC_BITBUCKET_CLIENT_ID: string = process.env.NEXT_PUBLIC_BITBUCKET_CLIENT_ID as string;
+const BITBUCKET_CLIENT_SECRET: string = process.env.BITBUCKET_CLIENT_SECRET as string;
+
+export default async function exchangeBitBucketCodeForTokensHandler(req: NextApiRequest, res: NextApiResponse): Promise<void> {
+  const { code }: { code: string } = req.body;
+
+  if (!code) {
+    return res.status(400).send("No authorization code provided");
```

Comment:
we're trying to unify api responses. to shed some light https://github.com/akash-network/console/blob/main/apps/api/src/core/services/hono-error-handler/hono-error-handler.service.ts

Basically it should replicate `http-errors` serialised error

```suggestion
    return res.status(400).send({ error: 'BadRequestError', message: "No authorization code provided",  });
```